content/docs/prometheus-metrics/_index.md

```diff
@@ -6,9 +6,9 @@ weight: 7
 draft: false
 ---
 
-ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be disabled with the `--disable-metrics` command-line flag or the `DISABLE_METRICS` environment variable.
+ChartMuseum exposes its [Prometheus metrics](https://prometheus.io/docs/concepts/metric_types/) at the `/metrics` route on the main port. This can be enabled with the `--enable-metrics` command-line flag or the `ENABLE_METRICS` environment variable.
 
-> Note that the Kubernetes chart currently disables metrics by default (`DISABLE_METRICS=true` is set in the chart).
+> Note that the Kubernetes chart currently disables metrics by default (`ENABLE_METRICS=false` is set in the chart). The `--disable-metrics` command-line flag has be deprecated and will only be available in `v0.14.0` and prior.
```

Comment:
We should mention that it’s disabled by default in general and not only disabled in the chart.

Reply:
okay will update it now